packages/jasmine/src/adapter/JasmineAdapter.ts

```diff
@@ -80,6 +80,13 @@ export class JasmineAdapter implements TestRunnerAdapter {
         ));
 
         this.runner.addReporter(reporter(this.runner.jasmine));
+
+        if (this.config.additionalReporters){
+            const additionalReporters = this.config.additionalReporters
+            console.log(`${additionalReporters.length} Additional reporter(s) found. Going to configure them...`)
```

Comment:
Would you mind removing `console.log` so that the output doesn't get printed when we run our own unit and integration tests?